src/views/templates/list/hooks/useWorkloadProfile.ts

```diff
@@ -0,0 +1,25 @@
+import { V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { getTemplateWorkload } from '@kubevirt-utils/resources/template';
+
+type WorkloadLabelsProps = {
+  desktop: string;
+  server: string;
+  highperformance: string;
+};
+
+const useWorkloadLabels = (): WorkloadLabelsProps => {
+  const { t } = useKubevirtTranslation();
+
+  return {
+    desktop: t('Desktop'),
+    server: t('Server'),
+    highperformance: t('High performance'),
+  };
+};
+
+const useWorkloadProfile = (template: V1Template): string => {
+  return useWorkloadLabels()?.[getTemplateWorkload(template)];
```

Comment:
in the list view you sort by a simple `object[0]...` to get the workload, here you use a method, this is an opening for bugs in cases the `getTemplateWorkload`  will get a different workload.